setup.cfg

```diff
@@ -1,33 +1,85 @@
-[build_sphinx]
-source-dir = docs
-build-dir = docs/_build
-all_files = 1
-
-[upload_docs]
-upload-dir = docs/_build/html
-show-response = 1
-
-[tool:pytest]
-addopts = --doctest-ignore-import-errors
-minversion = 3.0
-norecursedirs = build docs/_build
-doctest_plus = enabled
-
-[ah_bootstrap]
-auto_use = True
-
 [metadata]
-package_name = astroplan
-description = Observation planning package for astronomers
-long_description = Observation planning package for astronomers
+name = astroplan
 author = Astroplan developers
 author_email = astropy-dev@googlegroups.com
 license = BSD
+license_file = LICENSE.rst
 url = https://github.com/astropy/astroplan
+description = Observation planning package for astronomers
+long_description = file: README.rst
+long_description_content_type = text/x-rst
 edit_on_github = False
 github_project = astropy/astroplan
 
-[entry_points]
+[options]
+zip_safe = False
+packages = find:
+python_requires = >=3.6
+setup_requires = setuptools_scm
+install_requires =
+    numpy>=1.10
+    astropy>=1.3
```

Comment:
@bmorris3 @wtgee  -- Can the astropy and/or numpy minimum versions be updated? I'd suggest something more like `numpy>=1.17` and `astropy>=4` though maybe astropy 3 works.

Reply:
yeah, since 4.0.x is the new LTS, i hard agree that these should be updated.